index.html

```diff
@@ -1,16 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel='stylesheet' id='cookie-consent-style-css'  href='https://www.munroandpartners.com/wp-content/plugins/uk-cookie-consent/assets/css/style.css?ver=4.9.6' type='text/css' media='all' />
+    <link rel='stylesheet' id='wp-pagenavi-css'  href='https://www.munroandpartners.com/wp-content/plugins/wp-pagenavi/pagenavi-css.css?ver=2.70' type='text/css' media='all' />
+    <link rel='stylesheet' id='Divi-css'  href='https://www.munroandpartners.com/wp-content/themes/Divi/style.css?ver=4.9.6' type='text/css' media='all' />
+    <link rel='stylesheet' id='divi-fonts-css'  href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800&#038;subset=latin,latin-ext' type='text/css' media='all' />
+    <link rel='stylesheet' id='divi-style-css'  href='https://www.munroandpartners.com/wp-content/themes/sift/style.css?ver=3.6' type='text/css' media='all' />
+    <link rel='stylesheet' id='et-builder-googlefonts-cached-css'  href='https://fonts.googleapis.com/css?family=Raleway%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2Cregular%2Citalic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic%7CPoppins%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2Cregular%2Citalic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic&#038;ver=4.9.6#038;subset=latin,latin-ext,devanagari' type='text/css' media='all' />
+    <link rel='stylesheet' id='dashicons-css'  href='https://www.munroandpartners.com/wp-includes/css/dashicons.min.css?ver=4.9.6' type='text/css' media='all' />
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>calculators-simple</title>
     <!-- this is just to demo the component style mixin -->
     <style>
-      .et_btn {
-        color: red
-      }
       .btn {
-        color: blue
+        color: #ffffff;
+        padding: 1em 5em;
+        font-size: 1rem;
+        font-weight: bold;
```

Comment:
Font weight bold and font weight 700 are both correct and ok to use. As we use google fonts which define font weights I always prefer to use 700. (As different fonts may have other values e.g 600 etc)

Reply:
Changed bold to 700 due too best practice.